packages/schema/src/idl.rs

```diff
@@ -22,7 +21,7 @@ pub struct Api {
     pub migrate: Option<RootSchema>,
     pub sudo: Option<RootSchema>,
     /// A mapping of query variants to response types
-    pub responses: BTreeMap<String, RootSchema>,
+    pub responses: Option<BTreeMap<String, RootSchema>>,
```

Comment:
Is there a difference between None and an empty map? None when no query endpoint exists?

Reply:
Yes, `None` when there's no query endpoint. I could improve it so that both `query` and `responses` are behind one `Option` if you really want me to.

Reply:
OK, I'm keeping it as is unless someone needs this to change.